### PR TITLE
[status] List of configured endpoints move visible in the status page

### PR DESCRIPTION
**Pull request by remeh**
### What does this PR do?

Adds an "Endpoints" section in the Agent GUI status page and in the CLI status command.

### Motivation

Endpoints information was deducible from the API key validity lines, however it was not clear, this is why we want a dedicated section.

### Additional Notes

Removed the endpoint info from the API key validity line.

### CODE REVIEW

**Comment by arbll** [comment]
Can you add a screen of the changes ?  🖼

**Comment by remeh** [comment]
Of course:

capture d ecran 2018-12-05 a 17 54 45

I've also tried a list for the api keys under each endpoint, but I wasn't convinced by the result. Also, most of the case they will be only one api key.
But let me know if you think it's better.

**Review by hush-hush — CHANGES_REQUESTED** [review]
Sadly we have 2 files for the status page: one for the GUI in HTML and one in pure text for the CLI status page.

This means you have to duplicate the change from `cmd/agent/gui/views/templates/generalStatus.tmpl` to `pkg/status/dist/templates/forwarder.tmpl` (or add another section).

File: pkg/status/status.go [hunk 1]
```diff
@@ -184,6 +184,27 @@ func getPartialConfig() map[string]string {
 	return conf
 }
 
+func getEndpointsInfos() (map[string]interface{}, error) {
+	endpoints, err := config.GetMultipleEndpoints()
+	if err != nil {
+		return nil, err
+	}
+
+	rv := make(map[string]interface{})
```
[the remark this answers is not in the document]
Comment (remeh) [reply pkg/status/status.go]: Yep sorry, old habit (rv = returned value), thanks for the 👀

File: cmd/agent/gui/views/templates/generalStatus.tmpl [hunk 2]
```diff
@@ -165,6 +165,23 @@
...
   </div>
 
+  <div class="stat">
+    <span class="stat_title">Endpoints</span>
+    <span class="stat_data">
+    {{- with .endpoints}}
+      {{- range $key, $value := .}}
+        {{$key}}
+        {{- range $idx, $apikey := $value }}
+         - Api key ending with: {{$apikey}}
```
[the remark this answers is not in the document]
Comment (remeh) [reply cmd/agent/gui/views/templates/generalStatus.tmpl]: Consider it done

**Comment by remeh** [comment]
Update the GUI rendering and added the display in the cli status cmd:

capture d ecran 2018-12-06 a 11 22 21

capture d ecran 2018-12-06 a 11 22 09

**Review by hush-hush — DISMISSED** [review]
:ok_hand:

**Review by xvello — COMMENTED** [review]
LGTM 👍

**Review by hush-hush — APPROVED** [review]

**Pull request merged**